File: MES_EQUIPMENT/Models/ProcessLogDto.cs
public class ProcessLogDto
{
    public int ProcessLogId { get; set; }
    public int WorkOrderId { get; set; }
    public int LotId { get; set; }
    public int MemberId { get; set; }
    public int EquipmentId { get; set; }
    public DateTime StartTime { get; set; }
    public DateTime EndTime { get; set; }

    // ✅ 역직렬화: byte[] → DTO
    public static ProcessLogDto FromBytes(byte[] payload)
    {
        int offset = 0;
        ProcessLogDto dto = new ProcessLogDto();

        dto.ProcessLogId = BitConverter.ToInt32(payload, offset); offset += 4;
        dto.WorkOrderId = BitConverter.ToInt32(payload, offset); offset += 4;
        dto.LotId = BitConverter.ToInt32(payload, offset); offset += 4;
        dto.MemberId = BitConverter.ToInt32(payload, offset); offset += 4;
        dto.EquipmentId = BitConverter.ToInt32(payload, offset); offset += 4;

        long startBinary = BitConverter.ToInt64(payload, offset); offset += 8;
        long endBinary = BitConverter.ToInt64(payload, offset); offset += 8;

        dto.StartTime = DateTime.FromBinary(startBinary);
        dto.EndTime = DateTime.FromBinary(endBinary);

        return dto;
    }
}

File: MES_SPRING/src/main/java/com/hm/mes_final_260106/dto/ProcessLogDto.java
package com.hm.mes_final_260106.dto;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.time.LocalDateTime;

@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class ProcessLogDto {

    private Long id;            // PK

    // --- FK (Foreign Keys) ---
    // DB의 snake_case(_id)를 Java의 camelCase(Id)로 변환
    private Long workOrderId;   // WorkOrder 참조
    private Long lotId;         // Lot 참조
    private Long memberId;      // Member 참조
    private Long equipmentId;   // Equipment 참조

    // --- Time Fields ---
    private LocalDateTime startTime; // 시작 시간
    private LocalDateTime endTime;   // 종료 시간
}
///1111111111